### DISCUSSION TITLE
Is there a recommended way to track progress in books that are release incrementally?

### DISCUSSION THREAD

**Discussion by LivInTheLookingGlass** [bookwyrm-social/bookwyrm#1280] [Q&A]
Examples of these include:

- Webnovels (like The Martian)
- Comics, Manga, and similar which are released 1 chapter or 1 volume at a time

It seems to me that the currently implemented progress tracking might not suit these use cases well, because it assumes you know how long the full story is

**Comment by mouse-reeve** [comment]
There isn't a recommended way - right now we just have page number and percent, which you're right to observe don't fit serialized online writing well.
